### PR TITLE
CDPS-1054: Renamed template project to hmpps-person-integration-api

### PR DESCRIPTION
**Pull request by mtac50**

### CODE REVIEW

**Review by brightonsbox — APPROVED** [review]
LGTM

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -17,4 +17,4 @@ generic-service:
 # CloudPlatform AlertManager receiver to route prometheus alerts to slack
 # See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
 generic-prometheus-alerts:
-  alertSeverity: NON_PROD_ALERTS_SEVERITY_LABEL
+  alertSeverity: digital-prison-service-dev
```
Comment (brightonsbox) [inline helm_deploy/values-dev.yaml:20]: actually check line 8 above - ingress still `template-kotlin-dev.hmpps.service.justice.gov.uk`

Comment (mtac50) [reply helm_deploy/values-dev.yaml:20]: Good spot, thanks!

**Review by brightonsbox — APPROVED** [review]
LGTM

**Pull request merged**